catalog/USGS/USGS_NLCD_RELEASES_2021_REL_CANOPY_YEAR.jsonnet

```diff
@@ -0,0 +1,240 @@
+// The asset id as referenced in Earth Engine:
+// ee.Image('projects/usgs-gee-audra-griebel/assets/eros/canopy/year')
+
+local id = 'USGS/NLCD_RELEASES/2021_REL/CANOPY/YEAR';
+local subdir = 'USGS';
+
+local ee_const = import 'earthengine_const.libsonnet';
+local ee = import 'earthengine.libsonnet';
+local spdx = import 'spdx.libsonnet';
+local units = import 'units.libsonnet';
+
+local license = spdx.cc0_1_0;
+
+local basename = std.strReplace(id, '/', '_');
+local self_ee_catalog_url = ee_const.ee_catalog_url + basename;
+
+{
+  // TODO(schwehr): Remove skip_indexing when the dataset is ready.
+  'gee:skip_indexing': true,
+  'gee:user_uploaded': true,
+
+  stac_version: ee_const.stac_version,
+  type: ee_const.stac_type.collection,
+  stac_extensions: [
+    ee_const.ext_eo,
+    ee_const.ext_sci,
+    ee_const.ext_ver,
+  ],
+  id: id,
+  title: 'Tree Canopy Cover (TCC) data (2011-2021)' + version,
+  version: 'version',
+
+  // TODO(schwehr): what if it is an image collection
+  'gee:type': ee_const.gee_type.imagecollection,
+
+  description: |||
+    The USDA Forest Service (USFS) builds two versions of percent tree canopy cover data, 

```

Comment:
Fit to 80 columns

Reply:
I can't ever see the column width. Is there a button to show that? 
I believe I got this fixed (I copied the raw from the test file so it should be right).